frontend/vue/components/UserAccount/PrivacySection.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <section class="privacy-section">
+    <h3 class="privacy-section__title">
+      Privacy
+    </h3>
+    <h4 class="privacy-section__subtitle privacy-section__subtitle_general">
+      IBM Privacy Statement
+    </h4>
+    <AppLink class="privacy-section__policy-link" url="https://www.ibm.com/privacy">
+      https://www.ibm.com/privacy
+    </AppLink>
+    <h4 class="privacy-section__subtitle privacy-section__subtitle_quantum">
+      IBM Quantum End User Agreement
+    </h4>
+    <!-- eslint-disable-next-line vue/no-v-html -->
+    <div class="privacy-section__policy-text" v-html="privacyPolicyHTML" />
+  </section>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue-demi'
+import MarkdownIt from 'markdown-it'
+import axios, { AxiosResponse } from 'axios'
+import AppLink from '../common/AppLink.vue'
+
+export default defineComponent({
+  name: 'PrivacySection',
+  components: {
+    AppLink
+  },
+  data () {
+    return {
+      options: {
+        html: true,
+        xhtmlOut: true,
+        breaks: true,
+        linkify: true
+      },
+      privacyPolicyMd: 'Loading...'
+    }
+  },
+  computed: {
+    markdownItInstance (): MarkdownIt { return MarkdownIt(this.options) },
+    privacyPolicyHTML (): string { return this.markdownItInstance.render(this.privacyPolicyMd) }
+  },
+  mounted () {
+    axios.get('/markdown/privacy-policy.md').then(
+      (response: AxiosResponse<string>) => {
+        this.privacyPolicyMd = response.data
+      },
+      () => {
+        this.privacyPolicyMd = 'Error loading privacy policy text.'
+      }
+    )
+  }
+})
+</script>
+
+<style lang="scss" scoped>
+@import 'carbon-components/scss/globals/scss/typography';
+@import '~/../scss/variables/mq.scss';
+@import '~/../scss/variables/colors.scss';
+@import '~/../scss/mixins/mixins.scss';
+
+.privacy-section {
+  @include contained();
+  margin-left: 0;
+  padding-top: $spacing-07;
+  padding-bottom: $spacing-07;
+
+  &__title {
+    @include type-style('expressive-heading-04');
+    color: $text-active-color;
+    border-bottom: 1px solid $border-color-light-2;
+    padding-bottom: $spacing-05;
+  }
+
+  &__subtitle {
+    @include type-style('expressive-heading-04');
+    padding: $spacing-05 0;
+
+    &_quantum {
+      margin-top: $spacing-08;
+    }
+  }
+
+  &__policy-link {
+    @include type-style('body-long-02');
+    color: $link-color-secondary;
+  }
+
+  &__policy-text {
+    padding: $spacing-05;
```

Comment:
I believe `$spacing-06` might be closer to the design:

```suggestion
    padding: $spacing-06;
```